docs/src/DeveloperDocumentation/printing_details.md

```diff
@@ -16,12 +16,12 @@ e.g. to print the `base_ring(X)` when in `one line` mode.
 It exists to make sure that `one line` stays compact and human readable.
 
 Top-level REPL printing of an object will use `detailed` mode by default
-```julia
+```julia-repl
 julia> X
 detailed
 ```
 Inside nested structures, e.g. inside a `Vector`, the `one line` mode is used.
-```julia
+```julia-repl
 julia> [X,X]
 3-element Vector{TypeofX{T}}
 one line
```

Comment:
unrelated to the change here, but could you indent the `one line` thingies by one space each, such that it looks like how julia shows vectors?

Reply:
I can have a look...

but grepping for this text just made me aware that we duplicate quite a bit between
- `docs/src/General/printing.md` and
- `docs/src/DeveloperDocumentation/printing_details.md`

including this text... Hmmm